retrofit/src/main/java/retrofit2/Retrofit.java

```diff
@@ -53,22 +53,45 @@
  * Response&lt;User&gt; user = api.getUser().execute();
  * </code></pre>
  *
+ * <h3>Customize your rest adapter with newBuilder()</h3>
```

Comment:
What's a "rest adapter"?!? I don't think we want any of this doc. Unlike OkHttp, this isn't something we expect many to do (seeing as we haven't had the ability for Retrofit's 5 years of existence).